.github/ISSUE_TEMPLATE

```diff
@@ -1,15 +1,27 @@
-Have you read the guidelines for contributing?
-
 Before opening an issue, please read the guidelines for contributing linked above. Issues with invalid formats, or issues not mentioning the maintainer of the applet, may not get a response, and eventually will be closed. You can find the author by visiting the applet's page on the Spices website (https://cinnamon-spices.linuxmint.com/), or going to the applet's directory on this repository, and checking the the author field in info.json. If it is set to none in info.json, or there is no author listed on the site, then this step is not necessary.
 
-[Notify author of applet]: @<GithubAuthor>
+```
+ * Notify author of applet @<GithubAuthor>
```

Comment:
to be found here: https://github.com/linuxmint/cinnamon-spices-applets/blob/master/.github/CODEOWNERS